setup.py

```diff
@@ -11,5 +11,6 @@
       install_requires=[
         'websocket-client',
         'requests',
+        'six',
```

Comment:
Mind adding `six` to [requirements.txt](https://github.com/slackhq/python-slackclient/blob/master/requirements.txt) as well?